ci-scripts/setup-cluster.sh

```diff
@@ -358,6 +360,182 @@ $CONDITIONAL_FIELDS
     server_port: 8080
     prometheus_port: 9090
 EOF
+      else
+        oc create secret generic logging-loki-s3 \
+  --from-literal=bucketnames="${AWS_BUCKET_NAME}" \
+  --from-literal=endpoint="https://s3.eu-west-1.amazonaws.com" \
+  --from-literal=region="eu-west-1" \
+  --from-literal=access_key_id="${AWS_ACCESS_ID}" \
+  --from-literal=access_key_secret="${AWS_SECRET_KEY}"
+
+        oc create namespace openshift-operators-redhat
+
+        # Create OperatorGroup for installing loki-operator
+        oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-operators-redhat
+EOF
+
+
+        # Create Subscription for installing Loki Operator
+        oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: loki-operator
+  namespace: openshift-operators-redhat
+spec:
+  channel: stable-6.0
+  name: loki-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+
+EOF
+        sleep 60
+
+        echo "Checking for InstallPlan..."
+        INSTALL_PLAN=$(oc get installplan -n openshift-operators -o json | jq -r '.items[] | select(.spec.approved == false) | .metadata.name')
+
+        if [[ -n "$INSTALL_PLAN" ]]; then
+          echo "Approving InstallPlan: $INSTALL_PLAN"
+          oc patch installplan $INSTALL_PLAN -n openshift-operators --type='merge' -p '{"spec":{"approved":true}}'
+        fi
+
+        sleep 240
```

Comment:
I think we can use the following label based wait instead of sleep:
`wait_for_entity_by_selector 300 openshift-operators-redhat pod name=loki-operator-controller-manager`